src/nvim/version.c

```diff
@@ -57,7 +57,6 @@ static char *(features[]) = {
   "+cursorbind",
   "+cursorshape",
   "+dialog_con",
-  "+diff",
```

Comment:
Perhaps `Features included (+) or not (-):` could be changed for `Includes support for:`.

Also, a list of removed features would be good to have, to make clean how nvim differs from vim: `-clipboard`, `-clientserver` `-cryptv`, etc. Just as an high level overview, there is no point in being too detailed.